src/js/components/Diagram/index.d.ts

```diff
@@ -17,10 +17,16 @@ export type DiagramAnimationType =
 
 export interface DiagramProps {
   animation?: DiagramAnimationType;
+  arrow?: {
```

Comment:
Hmm... there's a lot of potential variety in arrowhead styling. I think we should have our first pass at arrows not include any styling customization. We should pick one way to start with, use the color for the connection, and use the thickness for the connection to drive both the thickness of the arrowhead lines as well as their length. So, thinner lines get smaller arrowheads. I think we should start with something along these lines and consider further arrowhead customization at a later time.

Reply:
Sounds, good.
Can you please, tell me know which arrowhead you are particularly refering to in these screenshots ? https://github.com/grommet/grommet/pull/5950#issue-1138842626

Reply:
Let's start with just the open arrow point, with just two legs, like is shown in the screenshot in the comment above.